.rubocop.yml

```diff
@@ -0,0 +1,12 @@
+AllCops:
+  RunRailsCops: true
+  Exclude:
+    - 'bin/**/*'
+    - 'test/**/*'
+    - 'config/**/*'
+    - 'Gemfile'
+    - 'db/schema.rb'
+    - 'db/seeds.rb'
```

Comment:
Sure, but I didn't editied db/seeds.rb so I've excluded this. In other case it would be left for rubocop check.